File: vue/src/api/log.js
import request from '@/utils/request'

export function getLog(data) {
  return request({
    url: '/log/list',
    method: 'post',
    data: data
  })
}

export function searchLog(data) {
  return request({
    url: '/search/log',
    method: 'post',
    data: data
  })
}

export function logClassify() {
  return request({
    url: '/log/classify',
    method: 'post'
  })
}

File: go/routegroup/system/log.go
package system

import (
	"itflow/handle"
	"itflow/internal/log"

	"github.com/hyahm/xmux"
)

// Log 日志路由
var Log *xmux.RouteGroup

func init() {
	Log = xmux.NewRouteGroup()

	Log.Post("/search/log", handle.SearchLog).BindJson(&log.SearchLog{})

	Log.Post("/log/classify", handle.LogClassify)

	// 	Log.Post("/log/list", handle.LogList).BindJson(&log.SearchLog{})
}
